handlers.js

```diff
@@ -0,0 +1,66 @@
+var fs          = require('fs');
+var url         = require('url');
+var querystring = require('querystring');
+
+var static 		= require('node-static');
```

Comment:
Using node-static made the code much more readable/friendly.

Happy to change it to be without though - let me know!